Configs/Config.ps1

```diff
@@ -121,4 +121,5 @@ $config.vds = @{
         linkproto = [string]'LLDP'
         linkoperation = [string]'Both'
         mtu = [int]1500
+        uplinkCount = [int]8
```

Comment:
You'll need to add `uplinkCount` to the list of $VDSKeys in line 110 of the Config.Tests.ps1 file.